docs/cse/administration/custom-inventory-sources.md

```diff
@@ -294,10 +287,11 @@ _`sourceCategory="security/jamf" and _collector="Jamf" \
 "customInventory": true,
 "type": "user"
 }
-
+```
 
 ### Webhook payload for Computer Entity
 
+```
```

Comment:
Code below and above are JSON, so I would enable syntax highlighting for them. 

```suggestion
```json
```